flink-connectors/flink-connector-hbase-1.4/src/test/java/org/apache/flink/connector/hbase1/HBaseDynamicTableFactoryTest.java

```diff
@@ -181,6 +185,25 @@ public void testTableSinkFactory() {
                 new DataType[] {DECIMAL(10, 3), TIMESTAMP(3), DATE(), TIME()},
                 hbaseSchema.getQualifierDataTypes("f4"));
 
+        // verify hadoop Configuration
+        org.apache.hadoop.conf.Configuration expectedConfiguration =
+                HBaseConfigurationUtil.getHBaseConfiguration();
+        expectedConfiguration.set(HConstants.ZOOKEEPER_QUORUM, "localhost:2181");
+        expectedConfiguration.set(HConstants.ZOOKEEPER_ZNODE_PARENT, "/flink");
+        expectedConfiguration.set("hbase.security.authentication", "kerberos");
+        Map<String, String> expectedProperties =
+                Lists.newArrayList(expectedConfiguration.iterator()).stream()
+                        .collect(Collectors.toMap(Map.Entry::getKey, Map.Entry::getValue));
```

Comment:
We can directly construct the expected Map.

Reply:
solved